swift/ql/lib/codeql/swift/frameworks/StandardLibrary/NsUrl.qll

```diff
@@ -10,6 +10,6 @@ private import codeql.swift.dataflow.ExternalFlow
  */
 private class NsUrlSummaries extends SummaryModelCsv {
   override predicate row(string row) {
-    row = ";NSURL;true;init(string:);(String);;Argument[0];ReturnValue;taint"
+    row = ";NSURL;true;init(string:);(String);;Argument[0];ReturnValue.OptionalSome;taint"
```

Comment:
Interesting, didn't know you could have .OptionalSome in the CSV. Do you know how this string label "OptionalSome" is related to the actual implementation of the `[some:0]` ContentSet?

Reply:
Yes, there's a special case in `parseContent` for `OptionalSome`, it's just syntactic sugar for `EnumElement[some:0]` (only cleaner and hopefully less error prone).